### ISSUE TITLE
Casters would like to have powered down structures be more clear

### ISSUE THREAD

**Issue by Ahli** [Ahli/HomeStoryCup-mod#9]
E.g. a tag on the building

**Comment by Peragore** [comment]
Yep! Having it look greyed out or a timer symbol on it or something like that would greatly increase glance value in the production tab

**Comment by Ahli** [comment]
Tag on the building has been added.

TODO add unpowered to production panel
